compatibility/bazel_tools/testing.bzl

```diff
@@ -52,13 +52,26 @@ excluded_test_tool_tests = [
     },
     {
         "start": "1.1.1",
+        "end": "0.0.0",  #FIXME: change to 1.3 after release
```

Comment:
@cocreature is there a better way to handle this?

Reply:
`0.0.0` is equivalent to just omitting `end`.

Reply:
actually is this really what you want? The end is inclusive. I think you want the latest snapshot at the time of merging.

Reply:
> I think you want the latest snapshot at the time of merging.
Right. I'll update that before merging.